### ISSUE TITLE
ods:status: unclear term, improve definition and add examples

### ISSUE THREAD

**Issue by DavidFichtmueller** [DiSSCo/openDS#163]
### Term Name

ods:status

### Digital Object Name

DigitalSpecimen

### Feedback

The name of this term is quite general and the definition doesn't help much to determine its purpose. Please improve the definition and add examples to make it more clear. Maybe also reference a controlled vocabulary, that could be used here.

**Comment by samleeflang** [comment]
Yes, I missed this in the generation of the terms pages (which we do based on schemas.dissco.tech). This term has a controlled vocabulary. I am adding all the controlled vocabularies into the Terms pages. Added an image on how the controlled vocabulary would look like in the Terms pages
![image](https://github.com/user-attachments/assets/b0d6039b-2894-48d4-bf6f-72ceea18dcd4)
